examples/default/main.c

```diff
@@ -47,6 +52,14 @@ int main(void)
     rtc_init();
 #endif
 
+#ifdef MODULE_GNRC
```

Comment:
Thinking about this -> I think gnrc support is not what we actually strive for here, is it? I think that actual function we want to include into the default example is simple link-layer access to any available network device (in the form of ifconfig and txtsend). Right now we have those implemented through the gnrc stack, but with netdev2 I see on the mid-term actually no need to include any gnrc specifics, rather I say we probably can make `txtsend` and `ifconfig` stack agnostic?!